### PR TITLE
Revision for CMS Kit Features naming

### LINKED ISSUE #14733: Revision for CMS Kit Features naming
image

* Remove all "Enabled" and "Enable" words here. They are completely unnecessary.
* Truly goup features. For example, there is no meaning to enable blog page without enabling the blogpost page. Blogging is a single feature and covers all blogging functionalities.
* The [feature definitions](https://github.com/abpframework/abp/blob/dev/modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Features/CmsKitFeatureDefinitionProvider.cs) should depend on the related global features. If a related global feature is disabled, these features should not be available at all.
* Write better descriptions for feature definitions. For example "Enable menu in the application" feels me like that if I disable that, tenants won't see any menu in the application. But actually, it is related to CMS Kit's dynamic menu system.

### PR DESCRIPTION
**Pull request by malikmasis**
fixes https://github.com/abpframework/abp/issues/14733

### CODE REVIEW

**Comment by malikmasis** [comment]
image

File: modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Localization/Resources/en.json [hunk 1]
```diff
@@ -194,23 +194,21 @@
...
-    "Feature:ReactionEnable": "Enabled reaction",
-    "Feature:ReactionEnableDescription": "Enable reaction in the application.",
-    "Feature:TagEnable": "Enabled taging",
-    "Feature:TagEnableDescription": "Enable taging in the application."
+    "Feature:BlogEnable": "Blogpost",
+    "Feature:BlogEnableDescription": "CMS Kit's blogpost system",
+    "Feature:CommentEnable": "Commenting",
+    "Feature:CommentEnableDescription": "CMS Kit's comment system",
+    "Feature:GlobalResourceEnable": "Global resourcing",
+    "Feature:GlobalResourceEnableDescription": "CMS Kit's dynamic globa resoruce system",
```
Comment (enisn) [inline modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Localization/Resources/en.json]:
```suggestion
    "Feature:GlobalResourceEnableDescription": "CMS Kit's global resoruces feature that allows managing global styles & scripts.",
```

File: modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Localization/Resources/en.json [hunk 2]
```diff
@@ -194,23 +194,21 @@
...
-    "Feature:RatingEnable": "Enabled rating",
-    "Feature:RatingEnableDescription": "Enable rating in the application.",
-    "Feature:ReactionEnable": "Enabled reaction",
-    "Feature:ReactionEnableDescription": "Enable reaction in the application.",
-    "Feature:TagEnable": "Enabled taging",
-    "Feature:TagEnableDescription": "Enable taging in the application."
+    "Feature:BlogEnable": "Blogpost",
+    "Feature:BlogEnableDescription": "CMS Kit's blogpost system",
+    "Feature:CommentEnable": "Commenting",
+    "Feature:CommentEnableDescription": "CMS Kit's comment system",
```
Comment (enisn) [inline modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Localization/Resources/en.json]:
```suggestion
    "Feature:CommentEnableDescription": "CMS Kit's comment system allows commenting on entities such as BlogPost.",
```

File: modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Localization/Resources/en.json [hunk 3]
```diff
@@ -194,23 +194,21 @@
...
+    "Feature:GlobalResourceEnable": "Global resourcing",
+    "Feature:GlobalResourceEnableDescription": "CMS Kit's dynamic globa resoruce system",
+    "Feature:MenuEnable": "Menu",
+    "Feature:MenuEnableDescription": "CMS Kit's dynamic menu system",
```
Comment (enisn) [inline modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Localization/Resources/en.json]:
```suggestion
    "Feature:MenuEnableDescription": "CMS Kit's dynamic menu system that allows adding/removing application menus dynamically.",
```

File: modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Localization/Resources/en.json [hunk 4]
```diff
@@ -194,23 +194,21 @@
...
+    "Feature:PageEnable": "Paging",
+    "Feature:PageEnableDescription": "CMS Kit's page system",
```
Comment (enisn) [inline modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Localization/Resources/en.json]:
```suggestion
    "Feature:PageEnableDescription": "CMS Kit's page system that allows creating static pages with specific URL.",
```

File: modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Localization/Resources/en.json [hunk 5]
```diff
@@ -194,23 +194,21 @@
...
+    "Feature:RatingEnable": "Rating",
+    "Feature:RatingEnableDescription": "CMS Kit's rating system",
```
Comment (enisn) [inline modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Localization/Resources/en.json]:
```suggestion
    "Feature:RatingEnableDescription": "CMS Kit's rating system that allows users to rate entities such as BlogPost.",
```

File: modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Localization/Resources/en.json [hunk 6]
```diff
@@ -194,23 +194,21 @@
...
+    "Feature:ReactionEnable": "Reaction",
+    "Feature:ReactionEnableDescription": "CMS Kit's reaction system",
```
Comment (enisn) [inline modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Localization/Resources/en.json]:
```suggestion
    "Feature:ReactionEnableDescription": "CMS Kit's reaction system that allows users to send reactions to entities such as BlogPost, Comments, etc.",
```

File: modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Localization/Resources/en.json [hunk 7]
```diff
@@ -194,23 +194,21 @@
...
+    "Feature:TagEnable": "Taging",
+    "Feature:TagEnableDescription": "CMS Kit's tag system"
```
Comment (enisn) [inline modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Localization/Resources/en.json]:
```suggestion
    "Feature:TagEnableDescription": "CMS Kit's tag system that allows tagging entities such as BlogPost."
```

File: modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Localization/Resources/en.json [hunk 8]
```diff
@@ -194,23 +194,21 @@
...
-    "Feature:PageEnable": "Enabled paging page",
-    "Feature:PageEnableDescription": "Enable paging page in the application.",
-    "Feature:RatingEnable": "Enabled rating",
-    "Feature:RatingEnableDescription": "Enable rating in the application.",
-    "Feature:ReactionEnable": "Enabled reaction",
-    "Feature:ReactionEnableDescription": "Enable reaction in the application.",
-    "Feature:TagEnable": "Enabled taging",
-    "Feature:TagEnableDescription": "Enable taging in the application."
+    "Feature:BlogEnable": "Blogpost",
+    "Feature:BlogEnableDescription": "CMS Kit's blogpost system",
```
Comment (enisn) [inline modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Localization/Resources/en.json]:
```suggestion
    "Feature:BlogEnableDescription": "CMS Kit's blogpost system that allows create blogs and posts dynamically in the application.",
```

**Review by enisn — CHANGES_REQUESTED** [review]

**Review by enisn — APPROVED** [review]

**Pull request merged**